packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -56,7 +56,7 @@ export interface IAreaChartAreaPoint {
   values: IAreaChartDataSetPoint;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33435&commitId=dfd9a66e05c17e6b1ddd59d72b9502d177872b3b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>